packages/GA4Client/src/commerce-handler.js

```diff
@@ -136,10 +136,147 @@ CommerceHandler.prototype.logCommerceEvent = function (event) {
             null;
     }
 
-    gtag('event', mapGA4EcommerceEventName(event), ga4CommerceEventParameters);
+    this.sendCommerceEventToGA4(
+        mapGA4EcommerceEventName(event),
+        ga4CommerceEventParameters
+    );
     return true;
 };
 
+CommerceHandler.prototype.sendCommerceEventToGA4 = function (
+    eventName,
+    eventAttributes
+) {
+    if (this.common.forwarderSettings.measurementId) {
+        eventAttributes.send_to = this.common.forwarderSettings.measurementId;
+    }
+
+    gtag('event', eventName, eventAttributes);
```

Comment:
@rmi22186 - wouldn't returning `this.sendCommerceEventToGA4` contradict the removal of returning true inside of the forEach loop on the above comments? let me know otherwise

Reply:
We may be misunderstanding each other.  A zoom might be easier.  Hit me up when you're online and we can pair through it!